tools/bigquery-to-xml/bq_to_xml.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+"""
+   Copyright 2020 Google LLC
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+"""
+
+import os
+import xml.etree.ElementTree as ET
+from xml.dom.minidom import parseString
+import lxml.etree as etree
+
+from json2xml import json2xml
+from google.cloud import bigquery
+
+
+def bigquery_to_xml(query, custom_root_node=False):
```

Comment:
Please add integration test w/ a public BQ dataset w/ nested schema (e.g. github nested `bigquery-public-data:samples.github_nested`)

Reply:
Added a test against the github_nested table